### ISSUE TITLE
Design Doc: Main components

### ISSUE THREAD

**Issue by RossMcConeghy** [TwoClunkers/Clunk-Genesis#15]
Define all avatar components including size and motion limits and origin offsets from the center of the collision capsule.

**Comment by RossMcConeghy** [comment]
![image](https://cloud.githubusercontent.com/assets/4841483/6887401/453c36fa-d613-11e4-8585-f6fe04d3897d.png)

**Comment by jesterstud** [comment]
:)  but if it is more than 1 unit wide, we gotta widen our walkway!

**Comment by RossMcConeghy** [comment]
Right! uhm... so apparently the Prefab was scaled down to 70% of that... I'll have to adjust the dimensions..

**Comment by jesterstud** [comment]
I'm gonna work on some editing tools for terrain to help us convince a team
of novices to help us with content.

**Comment by jesterstud** [comment]
It will be brilliant.
But now my wife wants me to build a chicken coop...

**Comment by RossMcConeghy** [comment]
[BotAvatarTemplate](/TwoClunkers/Clunk-Genesis/wiki/BotAvatarTemplate)

**Comment by RossMcConeghy** [comment]
Prelim model using these component heights and coordinates:
![image](https://cloud.githubusercontent.com/assets/4841483/6888947/41f84c90-d640-11e4-905c-dfae3468fa4c.png)
